linux/network_device_ops/pvrdma_network_device_ops.yml

```diff
@@ -86,29 +88,50 @@
           include_tasks: start_pvrdma_server.yml
 
         # Switch to client VM
-        - name: "Reset the client VM '{{ pvrdma_client_vm_name }}' for updating IP address"
+        - name: "Switch to client VM"
+          ansible.builtin.set_fact:
+            vm_name: "{{ pvrdma_client_vm_name }}"
+
+        - name: "Reset client VM '{{ pvrdma_client_vm_name }}' for updating IP address"
           include_tasks: ../../common/vm_set_power_state.yml
           vars:
-            vm_name: "{{ pvrdma_client_vm_name }}"
             vm_power_state_set: 'restarted'
 
-        - name: "Add client VM guest IP to inventory"
+        - name: "Get client VM's guest IP"
+          block:
+            - name: "Refresh client VM's guest IP till it has different IP with server VM"
+              include_tasks: refresh_pvrdma_client_vm_ip.yml
+              with_list: "{{ range(1, 11) | list }}"
+              loop_control:
+                loop_var: retry_count
```

Comment:
`retry_count` is not defined.

Reply:
It is defined by `loop_var`. Here uses a new var `retry_count` to distinguish the default loop var `item` in included tasks.

Reply:
I'm not seeing it's defined in the included task.

Reply:
Added a debug message for displaying the retry count:
```
TASK [Print retry count for refreshing client VM's guest IP] *******************
task path: /home/worker/workspace/Ansible_Cycle_SLES_15.x/ansible-vsphere-gos-validation/linux/network_device_ops/refresh_pvrdma_client_vm_ip.yml:9
ok: [localhost] => {
    "retry_count": 1
}
```

```
TASK [Print retry count for refreshing client VM's guest IP] *******************
task path: /home/worker/workspace/Ansible_Cycle_SLES_15.x/ansible-vsphere-gos-validation/linux/network_device_ops/refresh_pvrdma_client_vm_ip.yml:9
ok: [localhost] => {
    "retry_count": 2
}
```